docker/cloudbuild.yaml

```diff
@@ -4,45 +4,82 @@
 steps:
 # Build all supported versions.
 - name: 'gcr.io/cloud-builders/docker'
```

Comment:
Yeah, I considered making these concurrent, then didn't bother. Can add in a new PR if you want...